tests/property/ic_api/arg_data_raw/test/tests.ts

```diff
@@ -0,0 +1,109 @@
+import {
+    defaultPropTestParams,
+    expect,
+    getCanisterActor,
+    it,
+    Test
+} from 'azle/test';
+import { execSync } from 'child_process';
+import fc from 'fast-check';
+
+import { _SERVICE as Actor } from './dfx_generated/canister/canister.did';
+
+export function getTests(): Test {
+    return () => {
+        it('gets the init arg data raw from the canister', async () => {
+            await fc.assert(
+                fc.asyncProperty(
```

Comment:
Try to use arbitrary IDLs and Candid values if practical...use arbitrary bytes otherwise